server/events/policy_check_project_context_builder.go

```diff
@@ -17,14 +17,15 @@ func (p *PolicyCheckProjectContextBuilder) BuildProjectContext(
 	commentArgs []string,
 	repoDir string,
 	contextFlags *command.ContextFlags,
+	logLevel string,
 ) []command.ProjectContext {
-	prjCmds := p.ProjectCommandContextBuilder.BuildProjectContext(ctx, cmdName, prjCfg, commentArgs, repoDir, contextFlags)
```

Comment:
nit: do we need to pass in the log level to the policy check context builder since we only support this for plan and apply commands?

Reply:
i think the whole point of ContextFlags was to not have to modify these interfaces though.  for example, force apply is in there.  Let's just move this there.